Documentation/Books/Manual/Programs/Arangorestore/ParallelRestore.md

```diff
@@ -1,2 +1,198 @@
-Parallel Restore Procudure
+Parallel Restore Procedure
 ==========================
+
+The following parallel restore procedure is recommended to speed up the
+performance of [_arangorestore_](../Arangorestore/README.md) in a
+cluster environment.
+It is assumed that a cluster environment is running and a `dump` with
+[_arangodump_](../Arangodump/README.md) has been already been created.
+
+### Allocate dump Directory
+
+The first step is to copy the `dump` directory to all machines where
+coordinators are running. 
+
+{% hint 'info' %}
+This step is not strictly required as the backup can be restored over the
+network. However, if the restore is executed locally the restore speed is
+significantly improved.
+{% endhint %}
+	
+### Restore Collection Structure
+
+The collection structure has to be restored from exactly one coordinator (any
+coordinator can be used) with the following command:
+
+```
+arangorestore
+  --server.endpoint <endpoint-of-a-coordinator>
+  --server.database <database-name> 
+  --server.password <password> 
+  --import-data false 
+  --input-directory <dump-directory>
+```
+
+
+The option `--import-data false`  tells _arangorestore_ to restore only the
+collection structure and no data.
+
+### Set Replication Factor to 1
+
+It is necessary to set the the replication factor to 1 before importing any
+data. Run the following command from exactly one coordinator (any coordinator
+can be used):
+
+```
+echo 'db._collections().filter(function(c) { return c.name()[0] !== "_"; })
+.forEach(function(c) { print("collection:", c.name(), "replicationFactor:",
+c.properties().replicationFactor); c.properties({ replicationFactor: 1 }); });'
+| arangosh
+  --server.endpoint <endpoint-of-a-coordinator>
+  --server.database <database-name>
+  --server.username <user-name>
+  --server.password <password> 
+```
+
+### Create parallel restore scripts
+
+Now that the cluster is prepared, the parallel restore scripts have to be
+created. Therefore we will use a script that is provided in the following.
+
+parallelRestore.js:
+
+```
```

Comment:
I think there is a `sh` syntax highlighter

Should we mention that this will only work in bash?

Reply:
If it works only with bash you should ensure the use of bash and not use `/bin/sh` because that is like promising that it will run with any POSIX compatible shell and people will be annoyed if that is not the case. And bash scripts should not use `.sh` as extension. Usually the extension is omitted.

The official statement is:

Don't use extensions for your scripts. Scripts define new commands that you can run, and commands are generally not given extensions. Do you run ls.elf? Also: bash scripts are *not* sh scripts (so don't use .sh) and the extension will only cause dependencies headaches if the script gets rewritten in another language. See http://www.talisman.org/~erlkonig/documents/commandname-extensions-considered-harmful